etp-backend/src/main/clj/solita/etp/api/energiatodistus_crud.clj

```diff
@@ -41,7 +41,7 @@
                           :body save-schema}
              :access     (some-fn rooli-service/laatija? rooli-service/paakayttaja?)
              :responses  {200 {:body nil}
-                          404 {:body schema/Str}}
+                          404 {:body common-schema/GeneralError}}
```

Comment:
Muualla missä on tuota response-with-exceptions -mekanismia, näyttäisi että virhevastausta skemaa ei vain ole määrätty. Vaihtoehto voisi olla tässäkin jättää 404:n skema-speksi vain pois. Toisaalta jos se halutaan tähän, niin näitä voisi viljellä muuallekin.